package.json

```diff
@@ -21,7 +21,7 @@
     "build": "pnpm build:bundle && pnpm rebuild",
     "build:binding": "node-gyp build",
     "build:binding:debug": "node-gyp build --coverage --debug --verbose",
-    "build:bundle": "rimraf dist && tsdown -c tsdown.config.ts",
+    "build:bundle": "rimraf dist && tsdown -c tsdown.config.ts --sourcemap",
```

Comment:
Instead of setting `--sourcemap` here, we should set `sourcemap: true` in the `tsdown.config.ts` file.